.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Run unit tests
+
+on: [pull_request]
+
+jobs:
+  unit-tests:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        id: setup-python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+          architecture: x64
+      - name: Upgrade pip version
+        run: |
+          pip install --upgrade "pip>=21.3.1"
+      - name: Get pip cache dir
+        id: pip-cache
+        run: |
+          echo "::set-output name=dir::$(pip cache dir)"
+      - name: pip cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ${{ steps.pip-cache.outputs.dir }}
+            /opt/hostedtoolcache/Python
+            /Users/runner/hostedtoolcache/Python
+          key: ${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-pip-${{ hashFiles('**/setup.py') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-pip-
+      - name: Install dependencies
+        run: make install-ci-dependencies
+      - name: Install Feast
+        run : pip install feast==0.14.1
+      - name: Start local Trino cluster using Docker
+        run: make start-local-cluster
```

Comment:
BTW, I think we're planning on moving to use testcontianers for tests in feast tht require services where possible: https://github.com/testcontainers/testcontainers-python. The java version supports trino: https://www.testcontainers.org/modules/databases/trino/ , but i'm not sure about the python version. Maybe work looking into.